apis/v1alpha1/gatewayclass_types.go

```diff
@@ -85,10 +85,25 @@ type GatewayClassSpec struct {
 type GatewayClassConditionType string
 
 const (
-	// GatewayClassConditionStatusInvalidParameters indicates the
-	// validity of the Parameters set for a given controller. This
-	// will initially start off as "Unknown".
-	GatewayClassConditionStatusInvalidParameters GatewayClassConditionType = "InvalidParameters"
+	// GatewayClassConditionStatusAdmitted indicates that the GatewayClass
+	// has been accepted by the controller set in the `spec.controller`
```

Comment:
```suggestion
	// has been admitted by the controller set in the `spec.controller`
```

Reply:
Yeah, better not to be confusing.

Reply:
Quick sanity check here, this has been marked resolved but I'm still seeing "accepted" in the comments? Maybe my copy is out of date or an update didn't get pushed?